### PR TITLE
Cleanup3

### PR DESCRIPTION
**Pull request by maxbachmann**

### CODE REVIEW

**Comment by maxbachmann** [comment]
beside cleaning up this adds some todos aswell for parts that are broken

File: core/commons/CommonsManager.py [hunk 1]
```diff
@@ -82,10 +78,13 @@ def payload(message: MQTTMessage) -> dict:
 			p = p.decode()
 
 			if p == 'true' or p == 'false':
+				# TODO this is the same as return {str(p): p}, but less readable
+				# in this case even return {p: p} should be enough since it is already a string
 				raise ValueError
```
Comment (Psychokiller1888) [inline core/commons/CommonsManager.py]: Problem here are payloads differently structured, like payloads that are not key value based but only send a string per exemple

File: core/snips/samkilla/processors/ModuleRemoteProcessor.py [hunk 2]
```diff
@@ -86,6 +86,8 @@ def syncModulesOnAssistantSafely(self, typeEntityMatching: dict, moduleSyncState
 
 	# noinspection PyUnusedLocal
 	def syncModulesOnAssistant(self, typeEntityMatching: dict = None, moduleSyncState: str = None, hashComputationOnly: bool = False) -> tuple:
+		#TODO this appears wrong moduleSyncState is a str according to typing while self.createNewSavedModule() returns a dict
+		# is is used as a dict aswell. Is the typing wrong?
 		self._syncState = moduleSyncState or self.createNewSavedModule()
```
Comment (Psychokiller1888) [inline core/snips/samkilla/processors/ModuleRemoteProcessor.py]: Most likely an error when we strong typed everything? @jr-k ?

File: core/util/DatabaseManager.py [hunk 3]
```diff
@@ -211,7 +207,9 @@ def update(self, tableName: str, callerName: str, values: dict, query: str = Non
 			return True
 
 
-
+	#TODO this not really right, right now it either returns a list of sqlite3.Row when the method is not 'one',
+	# a sqlite2.Row or None when it is None and when there is an exception or no query sqlite3.Row() is a broken syntax
+	# -> will throw an TypeError (should probably return a empty list or None instead)
 	def fetch(self, tableName: str, query: str, callerName: str, values: dict = None, method: str = 'one') -> sqlite3.Row:
```
Comment (Psychokiller1888) [inline core/util/DatabaseManager.py]: Problem with returning an empty list is the typing then. Are you sure it is invalid? I made it a week ago to solve a problem

File: core/voice/LanguageManager.py [hunk 4]
```diff
@@ -150,6 +145,7 @@ def localize(self, string: str) -> str:
 
 
 	def changeActiveLanguage(self, toLang: str):
+		#TODO either typing is wrong or this string conversion is superfluous
 		toLang = str(toLang).lower()
```
Comment (Psychokiller1888) [inline core/voice/LanguageManager.py]: toLang.lower()

**Pull request merged**